halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -31,7 +31,7 @@
 
 @Data
 @EqualsAndHashCode(callSuper = true)
-public class KubernetesAccount extends ContainerAccount implements Cloneable {
+public class cKubernetesAccount extends ContainerAccount implements Cloneable {
```

Comment:
oh typo!

Reply:
Oops! The command key on my mac is going 😩 - fixed now